tests/Validator/CnpValidatorTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Tests\Validator;
+
+use App\Validator\Cnp;
+use App\Validator\CnpValidator;
+use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
+
+class CnpValidatorTest extends ConstraintValidatorTestCase
+{
+    public function testCnpIsNull(): void
+    {
+        $this->validator->validate(null, new Cnp());
+        $this->buildViolation('This is not a valid CNP')->assertRaised();
+    }
+
+    /**
+     * @dataProvider provideCnpValues
+     */
+    public function testCnp(string $cnp, bool $expected): void
+    {
+        $this->validator->validate($cnp, new Cnp());
+        if ($expected) {
+            $this->assertNoViolation();
+        } else {
+            $this->buildViolation('This is not a valid CNP')->assertRaised();
+        }
+    }
+
+    public function provideCnpValues(): array
+    {
+        return [
+            ['5010911070069', true, ['test']],
```

Comment:
It is strange.... testCnp requires 2 arguments.... 3 provided by privideCnpValues...